src/test/java/io/confluent/connect/hdfs/wal/FSWALTest.java

```diff
@@ -15,21 +15,27 @@
 
 package io.confluent.connect.hdfs.wal;
 
+import io.confluent.connect.hdfs.DataWriter;
```

Comment:
Most of our connectors use as import order as follows: 

```
import all other imports
<blank line>
import javax.*
import java.*
<blank line>
import io.confluent.*
<blank line>
import all other static imports
```
You might want to add this to you IDE to avoid a reshuffling as the one here.